services/QuillLMS/app/views/pages/ap.html.erb

```diff
@@ -3,4 +3,4 @@
 %>
 <%= render partial: 'pages/shared/explore_curriculum_navbar', active_tab: @active_tab  %>
 
-<%= react_component('Ap') %>
+<%= react_component('ApApp') %>
```

Comment:
ApApp!  😂

Reply:
Haha I had a giggle too when I was standardizing the naming